pkg/strategy/scmaker/strategy.go

```diff
@@ -5,22 +5,18 @@ import (
 	"fmt"
 	"math"
 	"sync"
-	"time"
 
 	log "github.com/sirupsen/logrus"
 
 	"github.com/c9s/bbgo/pkg/bbgo"
 	"github.com/c9s/bbgo/pkg/fixedpoint"
 	. "github.com/c9s/bbgo/pkg/indicator/v2"
-	"github.com/c9s/bbgo/pkg/risk/riskcontrol"
 	"github.com/c9s/bbgo/pkg/strategy/common"
 	"github.com/c9s/bbgo/pkg/types"
 )
 
 const ID = "scmaker"
 
-var ten = fixedpoint.NewFromInt(10)
```

Comment:
Is this expected to touch scmaker?

Reply:
yes, I cleaned up the scmaker and then copy scmaker to liquiditymaker